src/custom_modules/DAO_modules/donors.ts

```diff
@@ -39,8 +40,8 @@ async function getByID(ID): Promise<Donor | null> {
  * @param {Number} KID
  * @returns {Donor | null} A donor Object
  */
-async function getByKID(KID): Promise<Donor | null> {
```

Comment:
Is the return type now correctly inferred?

Reply:
Yes! 🚀

Reply:
⭐